examples/complete/main.tf

```diff
@@ -270,6 +271,9 @@ module "dedicated_host" {
 #############################################################################
 
 module "slz_vsi_dh" {
+  # depends_on                    = [module.dedicated_host]
+  # dedicated_host_id               = module.dedicated_host.dedicated_host_ids[0]
+  skip_iam_authorization_policy   = true
```

Comment:
why is this being skipped? This example is creating a key protect key and using it for encryption, so s2s auth policy is required right?

Reply:
I did not skipped it. This was already [part of the code](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/blob/77aed6e3c13f0f22f475b5e41570b437ba6e5c89/examples/complete/main.tf#L301), I just moved it up.

Reply:
Let me check this once again about the s2s part you mentioned.

Reply:
Removed `skip_iam_authorization_policy` which is not required as already `kms_encryption_enabled` is set as `true` in the same code and `existing_kms_instance_guid` too is provided as well.